packages/customer-erasure/bin/customer-erasure.js

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env node
+
+/* eslint-disable */
+/*
+  This file exists because we use Babel to transpile the JS but when testing
```

Comment:
General question: do we still need babel?

Reply:
I think babel is used for all/most CLIs? Maybe something to remove at some point in another step depending on the engine/node version we require?